api/tools/src/main/java/org/pmiops/workbench/tools/FetchWorkspaceDetails.java

```diff
@@ -9,20 +9,23 @@
 import org.apache.commons.cli.Options;
 import org.pmiops.workbench.db.dao.WorkspaceDao;
 import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.firecloud.FireCloudConfig;
+import org.pmiops.workbench.firecloud.FireCloudService;
+import org.pmiops.workbench.firecloud.FireCloudServiceImpl;
 import org.pmiops.workbench.firecloud.FirecloudTransforms;
 import org.pmiops.workbench.firecloud.api.WorkspacesApi;
 import org.pmiops.workbench.firecloud.model.FirecloudWorkspaceAccessEntry;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 
 /**
  * A tool that takes a Workspace namespace / Firecloud Project ID and returns details for any
  * workspaces found.
- *
- * <p>Details currently include... - Name - Creator Email - Collaborator Emails and Access Levels
  */
 @Configuration
+@Import({FireCloudServiceImpl.class, FireCloudConfig.class})
```

Comment:
Is it possible to pull in the WorkspaceAdminService (or other high-level service) instead? Some of this information is already being compiled for the admin page. There are a ton of transient dependencies though, so I can understand why you wouldn't.

Reply:
Maybe it's possible, but I don't think it would be preferable. I'm pulling in the smallest service that has the methods needed for this PR, which I think is the right thing to do, dependency-wise.